### ISSUE TITLE
exception for an already closed dialog

### ISSUE THREAD

**Issue by rjahn** [sibvisions/reactUI#491]
Use features/features, screen Contacts Error (DB). Select a record and an error popup is shown -> perfect.
Close with X or OK and another error is shown:

Component with id 'JVxMobileDemo_DP1_E1' was not found!

This shouldn't happen. Network communication debug view shows multiple close requests:

<img width="813" alt="image" src="https://github.com/sibvisions/reactUI/assets/1307043/3696f7ec-6e5e-4161-9bf6-fd8f1d7499d7">

Not sure why there are 2 close requests.

**Comment by gimmixAT** [comment]
I couldn't reproduce it in FF or Chrome but I added a check so the closeFrame request will be sent only one, when the dialog close handler is called multiple times for whatever reason. -> 98a6b28aa29a5e5f27a0599c1c95c6a4502eb681

**Comment by rjahn** [comment]
Chrome was my browser.

Above problem is fixed but the same problem occurs in Messages screen, button: Feedback
Press X in popup and not OK.

After X -> error appears:
<img width="621" alt="image" src="https://github.com/sibvisions/reactUI/assets/1307043/1b108dcc-4edb-4e0c-af3a-6600d577c07d">

BUT be careful, this is a problem, but there is another hidden bug (maybe):

After closing the above error, the next click (e.g. Feedback button) sends a request to the server and server sends a valid response, but our UI does nothing. If you press the button again, everything is fine.

So, please check why the response isn't handled after the error was shown before fixing the error.

**Comment by gimmixAT** [comment]
@rjahn just to double check, is the response for the `closeFrame` request supposed to look like this:

```json
[{"componentId":"Mes-TL","changedComponents":[{"~remove":"true","id":"IF3083"}],"name":"screen.generic","update":true,"home":false}]
```
especially that the id given in the ~remove doesn't relay to the componentId of the closed message

**Comment by gimmixAT** [comment]
There was an issue with reopening the same dialog which should be fixed now. As for the error message I still wasn't able to reproduce that, even in Chrome. Although I am wondering where the `_M6` comes from. In the requests I can see it is mostly `_M1` being used.

**Comment by rjahn** [comment]
_M6 looks like message 6 -> if M1, M2, M3, M4, M5 are not closed.

**Comment by rjahn** [comment]
M6 is the 6th message. I got the same message in my tests - after 6 times opening the same popup.
I will check this on server-side because it looks like a server numbering problem!

Following occurs in my environment:

https://github.com/sibvisions/reactUI/assets/1307043/bac10525-824d-4a6c-8f7d-bccd1a08c5ea

(Ignore first press-button)

My source is still before your fix for this problem - only to show you what's happening in my browser.
My primereact verson is 10.6.3, maybe things are different with 10.6.6?

**Comment by rjahn** [comment]
Works with your fix and primereact 10.6.6
